website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -0,0 +1,297 @@
+
+</detailsToggle>
+
+
+## When and where dbt Mesh is available
+
+<detailsToggle alt_header="Does dbt Mesh require me to be on a specific version of dbt?">
+
+Yes — your account must be on at least dbt v1.6 to take advantage of [cross-project dependencies](/docs/collaborate/govern/project-dependencies), one of the most crucial underlying capabilities required to implement a dbt Mesh.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Is there a way to leverage dbt Mesh capabilities in dbt Core?">
+
+Not all of them. While dbt Core provides some of the foundational elements for dbt Mesh, dbt Cloud offers an enhanced experience that leverages these elements for scaled collaboration across multiple teams. 
+
+Many of the key components that underpin dbt Mesh functionality, such as model contracts, versions, and access modifier levels (public and private), are available in dbt Core. To enable cross-project dependencies, users can also leverage [packages](/docs/build/packages). This enables users to import models from an upstream project, which allows the resolution of cross-project references.
+
+The major distinction comes with dbt Cloud's metadata service, which is unique to the dbt Cloud platform and allows for the resolution of references to only the public models in a project. This service enables users to take dependencies on resources from upstream projects without needing to load the full complexity of those upstream projects into their local development environment.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Does dbt Mesh require a specific dbt Cloud plan?">
+
+Yes, a [dbt Cloud Enterprise](https://www.getdbt.com/pricing) plan is required to set up multiple projects and reference models across them.
+
+</detailsToggle>
+
+## Tips on implementing dbt Mesh
+
+<detailsToggle alt_header="Is there a recommended migration or implementation process?">
+
+Refer to our developer guide on [How we structure our dbt Mesh projects](https://docs.getdbt.com/best-practices/how-we-mesh/mesh-1-intro). You may also be interested in watching the recording of this talk from Coalesce 2023: [Unlocking model governance and multi-project deployments with dbt-meshify](https://www.youtube.com/watch?v=FAsY0Qx8EyU).
+
+</detailsToggle>
+
+<detailsToggle alt_header="Are there tools available to help me migrate to a dbt Mesh?">
+
+`dbt-meshify` is a [CLI tool](https://github.com/dbt-labs/dbt-meshify) that automates the creation of model governance and cross-project lineage features introduced in dbt-core v1.5 and v1.6. This package will leverage your dbt project metadata to create and/or edit the files in your project to properly configure the models in your project with these features.
+</detailsToggle>
+
+<detailsToggle alt_header="TO DO -- What aspects of dbt Mesh are relevant to me?">
```

Comment:
needs answer filled in: [direct link to notion question](https://www.notion.so/WIP-dbt-Mesh-Customer-FAQ-bdadabc9917a4be697db53142c6ff979?pvs=4#7970a2e3eb174cfc96e4009b9133b096)